testing/src/cypress/page-objects/Institution-objects/ManageInstitutionObject.ts

```diff
@@ -1,4 +1,20 @@
-export default class ManageInstitutionObject {
+import BaseMethods from "./BaseMethods";
+
+export const enum SideBarMenuItems {
```

Comment:
This enum seems to be declared twice in the source code. Was it by mistake?

Reply:
Yes, fixed it